apps/react-native/babel.config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = function(api) {
+  api.cache(true);
```

Comment:
what was your decision making for the caching forever option?

Reply:
I need to take a look at this again.  I got an error/warning during development by not setting this.

Reply:
Set api.cache(false)